torch/fx/experimental/Partitioner.py

```diff
@@ -0,0 +1,145 @@
+from torch.fx import Graph, GraphModule
+from torch.fx import Node
+from torch.fx.experimental import GraphManipulation
+from typing import Dict, List, Tuple
+"""A graph module may not fit into one device.
```

Comment:
This comment is talking about partitioner not Partition, I'd move it to above the Partitioner class, and add a docstring to Partition describing what the class is used for.